### DISCUSSION TITLE
Position and naming of switches

### DISCUSSION THREAD

**Discussion by djmace** [OpenI6X/opentx#375] [Q&A]
I changed the hardware position of switches on my FlySky FS-I6X this way:

swa - 2pos
swb - 3pos
swc - 2pos
swd - a new installed 3pos

I am missing the ability to change the kind of switches in the "global hardware setup" under  categorie "switches".
FS.: for swb i have only the option off/button/2pos - but i would also need the option 3pos. Only swc gives me the option.

Is there any way to change this?

**Comment by ajjjjjjjj** [comment]
Any switch works as 3pos no matter the status in hardware.

**Reply by djmace** [reply]
Thank you, i already know that as it is functioning anyway. But my inner monk ;-) would be happier if the naming would reflect the kind of switch.

**Reply by ajjjjjjjj** [reply]
You're welcome. This is a known buttons mapping issue preventing doing changes in this menu. It's waiting for someone to pick it up, but due to cosmetic nature of this problem it is a low priority.

**Reply by djmace** [reply]
As long as it functioning a agree with you regarding to the priority. Thank you for explaining.
